src/Cookie/SetCookie.php

```diff
@@ -392,9 +392,10 @@ public function validate()
                 . 'following characters: ()<>@,;:\"/?={}';
         }
 
-        // Value must not be empty, but can be 0
+        // Value must not be null. 0 and empty string are valid. Empty strings
```

Comment:
These comments about zero are kinda stupid (not your fault). They must have been added after someone used empty by mistake instead of comparing against the empty string...

Reply:
I agree. But Im fine with keeping them.